packages/dds/tree/src/test/feature-libraries/chunked-forest/codec/schemaBasedEncoding.spec.ts

```diff
@@ -306,9 +306,11 @@ describe("schemaBasedEncoding", () => {
 
 				const context: FieldBatchEncodingContext = {
 					encodeType: TreeCompressionStrategy.Compressed,
+					originatorId: testIdCompressor.localSessionId,
 					schema: { schema: storedSchema, policy: defaultSchemaPolicy },
 					idCompressor,
 				};
+				idCompressor.finalizeCreationRange(idCompressor.takeNextCreationRange());
```

Comment:
are there any other snapshot tests that use identifiers that we should do something similar to?

Reply:
The only other place where I think we could do something similar are the `"identifier field encoding"` tests in `chunkEncodingEndToEnd.ts`, which is currently using the `testIdCompressor` (which always generated finalized ids). Should these be updated to use `createIdCompressor()`, and call `finalizeCreationRange` instead?

Reply:
No, I think this is fine. Let's get this in.